portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,40 +19,32 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.io.*;
-import java.util.*;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-	private ArrayList<String> msgs = new ArrayList<>();
-
 	@Override
 	public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-		String json = hardCodedJson();
+		ArrayList<String> msgs = new ArrayList<>();
```

Comment:
prefer to use List<> = Arraylist<>

It's good future proofing in case you ever want to change it to a different subclass